ubi-ruby-builder/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Ruby Builder
+FROM registry.access.redhat.com/ubi8/ubi
+ARG RUBY_MAJOR_VERSION
+ARG RUBY_FULL_VERSION
+ARG RUBY_SHA256=6c0bdf07876c69811a9e7dc237c43d40b1cb6369f68e0e17953d7279b524ad9a
+
```

Comment:
Why is this different from the other builder Dockerfiles? In particular, it's missing
```
# OpenSSL Builder
FROM openssl-builder:$OPENSSL_BUILDER_TAG as OpenSSL-builder
```

and

```
ENV LD_LIBRARY_PATH="/usr/local/ssl/lib"

COPY --from=OpenSSL-builder /usr/local/ssl/ /usr/local/ssl/

RUN sed -Ei 's/^# deb-src /deb-src /' /etc/apt/sources.list
```

Additionally, should the `UBI_VERSION` be an arg, the way that phusion and ubuntu versions are args in the other images?

Reply:
We can't use the openssl-builder because it compiled on top of ubuntu base image.
The UBI 8 image is already baked with FIPS openssl so we don't need to compile a dedicated openssl from ubi for that.
About `UBI_VERSION` it's not the same as ubuntu versioning, the ubi8/ubi repo will not change as the ubi 8 versions will be upgraded afaik.

Reply:
Added `UBI_VERSION` argument it my last commit :)